010-AzureDevOps/Coach/Solution-05.md

```diff
@@ -4,17 +4,4 @@
 
 ## Notes & Guidance
 
-This is the only section you need to include.
-
-Use general non-bulleted text for the beginning of a solution area for this challenge
-
-- Then move into bullets
-  - And sub-bullets and even
-    - sub-sub-bullets
-
-Break things apart with more than one bullet list
-
-- Like this
-- One
-- Right
-- Here
+- Check the solution file as ADO pipeline solution sample  (./solutions/solutions-05.yaml)
```

Comment:
Recommend making this a link with this format:

\[Display Text\]\(Path\)
Ex:
[Solution Link](./Solutions/Solution-05.yaml)